config/organise_runfolder.yml

```diff
@@ -0,0 +1,43 @@
+---
+#Input: runfolder_name
+
+#Currently named runfolderpath and runfolder to harmonize with rnaseq config by Monika, but for specific runfolder delivery, technically, these could be "merged" to one.
+runfolderpath: /proj/ngi2016001/incoming
+runfolder: <RUNFOLDERPATH>/<RUNFOLDER_NAME>
+organised: <RUNFOLDER>/Projects/
+
+#current path is /proj/ngi2016001/incoming/<RUNFOLDER>/Projects/<PROJECT>/<RUNFOLDER>/
+#The following is based on that we are to keep the same directory structure as above.
+files_to_organize:
+
+        #The fastq files
+        - source: <RUNFOLDER>/Unaligned/*  
+          destination: <ORGANISED>/(?P=projectid)/<RUNFOLDER_NAME>/Sample_(?P=samplename)/
+          options:
+                  required: True
+                  symlink: True
+                  regexp: (?P<projectid>[\w-]+)/Sample_(?P<samplename>[\w-]+)/(?P=samplename)_S(?P<samplenumber>\d+)_L(?P<lanes>\d+)_R(?P<read>\d)_001.fastq.gz
+
+        #The MultiQC files
+        - source: <RUNFOLDER>/seqreports/project/*
+          destination: <ORGANISED>/(?P=projectid)/<RUNFOLDER_NAME>/
+          options:
+                  required: True
+                  symlink: True
+                  regexp: (?P<projectid>[\w-]+)/(?P<RUNFOLDER_NAME>\w+)_(?P=projectid)_multiqc_report[\w.-]+
+
+
+
+        #what we are lacking, and what might need to be created outside the config is:
+        #1. checksums.md5
+        #2. Encrypted samplesheet
+        #As far as I know, these don't exist prior to organization.
+
+#        #hypothethical undetermined, needs an input of lane(s) connected to project
+#        #We will not include this at this point, as it requires additional input. Will maybe end up in its own config or something.
+#        - source: <RUNFOLDER>/Unaligned/Undetermined/Undetermined_S0_L<lanes>_R<read>_001.fastq.gz
+#          destination: <RUNFOLDER>/Unaligned/<PROJECT>/Undetermined/
+#          options:
+#                  required: False
+#                  symlink: True
+
```

Comment:
Indentation should be two spaces per level

Reply:
Fixed it!